phpcs.xml

```diff
@@ -1,15 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<ruleset name="drupal">
+<ruleset name="open_social">
     <description>PHP CodeSniffer configuration for Drupal coding standards in Open Social.</description>
-    <file>./modules/custom</file>
-    <file>./modules/social_features</file>
-    <file>./social.profile</file>
-    <file>./social.install</file>
-    <config name="ignore_errors_on_exit" value="0"/>
-    <config name="ignore_warnings_on_exit" value="0"/>
-    <config name="default_standard" value="Drupal"/>
+    <file>.</file>
```

Comment:
I think being explicit about our folders may be better here. What folders were we missing that are now included?

If someone runs `composer install` within Open Social (which is common in non-Drupal PHP projects) the new line will also check the `vendor` directory.

Reply:
It is missing `tests` and `src` and other `social.*` files. I will update this to exclude `translations.php`.

Reply:
`composer install` inside any module or profile is a developer mistake. This will break the whole project even if it is done inside any folder unless it is root, and not only PHPStan scans.